docs/frontend/debugging.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
I followed how the other docs are written, so I'm assuming this will generate the documentation correctly. I did not see a script to generate this on my local.

Reply:
Found the docs for doing building documentation locally, https://docs.dev.to/contributing/